semantic_conventions.yaml

```diff
@@ -0,0 +1,32 @@
+# This file describes standards and conventions for Span tags and logs.
```

Comment:
there are no links to this file - per earlier discussion maybe we should delete it altogether, so that .md file is the only source of truth?